src/world/draw.lua

```diff
@@ -60,4 +60,23 @@ function Draw.createPartDrawImageFunction(imageName)
 	end
 end
 
+function Draw.createDrawBlockDrawFunction(imageName)
+	local image = love.graphics.newImage("res/images/"..imageName..".png")
```

Comment:
Hm, headless tests will no longer pass with this `love.graphics` call.

Also, DrawBlockDrawFunction...why are there two "Draw"s in the name?